component/profile/BlogList.jsx

```diff
@@ -15,14 +15,14 @@ export default function BlogList({blogs}) {
       </div>
       <div className="flex items-start w-full overflow-x-auto pb-2">
         {blogs?.map((data) => (
-          <div
+          <div key={data._id}
             className={`flex items-start w-1/5 mt-3 justify-between pr-6 ${Profilestyles.min_w_9}`}
           >
             <div className="flex flex-col w-full">
-              {data?.image ? (
+              {data?.bannerImage ? (
```

Comment:
I don't think you need `?` here inside the loop...
`map` will look over an array only if an element exists in the array.